### DISCUSSION TITLE
GeckosIo server implementation in golang ?

### DISCUSSION THREAD

**Discussion by Arrognz** [geckosio/geckos.io#194] [General]
Hello, 

Thank you for geckosio, its really great !
I was wondering if it was imaginable to implement rtc server in golang and connect to it with geckosio client ?

**Comment by lulzsun** [comment]
I have started some work on re-implementing the server in golang and I am looking for contributors to help.

You can check it out here: [gecgos.io](https://github.com/lulzsun/gecgos.io)
